superdesk/default_settings.py

```diff
@@ -369,18 +419,15 @@ def env(variable, fallback_value=None):
 if not DEFAULT_TIMEZONE:
     raise ValueError("DEFAULT_TIMEZONE is empty")
 
-# The number of minutes since the last update of the Mongo auth object after which it will be deleted
+#: The number of minutes since the last update of the Mongo auth object after which it will be deleted
 SESSION_EXPIRY_MINUTES = int(env('SESSION_EXPIRY_MINUTES', 240))
 
-# The number of minutes before content items purged
-# akin.tolga 06/01/2014: using a large value (30 days) for the time being
-CONTENT_EXPIRY_MINUTES = int(env('CONTENT_EXPIRY_MINUTES', 4320))
```

Comment:
hey @akintolga this seems to be 3 days and not 30, so I've fixed the number, so if you want to keep it 3 days you should update settings in superdesk-aap to be sure after update